docs/index.md

```diff
@@ -27,7 +27,8 @@ The provider needs to be configured with the proper credentials before it can be
 terraform {
   required_providers {
     cdp = {
-      source = "registry.terraform.io/cloudera/cdp"
+      source = "cloudera-labs/cdp"
+      version = "0.1.2-pre"
```

Comment:
Do we need the version? Let's not include it here since otherwise every release we have to update the docs.

Reply:
I think this is only needed since our releases are pre-releases and have the `-pre` suffix after the semantic version. TF documentation mentions that to be able to use `x.y.z-aaa` type of releases you need to add the explicit version. I would say let's not document this yet, since hopefully we may have non-pre releases soon.

Reply:
Agreed.